roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -258,6 +258,20 @@
   tags:
     - bootstrap-os
 
+- name: Write cacert file
+  copy:
+    content: "{{ openstack_ca }}"
+    dest: "/etc/kubernetes/openstack-ca.pem"
+    group: "{{ kube_cert_group }}"
+    mode: 0640
+  when:
+    - inventory_hostname in groups['k8s-cluster']
+    - cloud_provider is defined
+    - cloud_provider in [ 'openstack', 'azure', 'vsphere' ]
+    - openstack_ca is defined
```

Comment:
Probably worth documenting this parameter somewhere. Maybe in kubespray-defaults?